code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -186,9 +186,16 @@
 	if(istype(blobmind) && istype(C))
 		blobmind.special_role = SPECIAL_ROLE_BLOB
 		var/obj/structure/blob/core/core = new(T, 200, C, 3)
-		if(core.overmind && core.overmind.mind)
-			//core.overmind.mind.name = blob.name
-			core.overmind.mind.special_role = SPECIAL_ROLE_BLOB_OVERMIND
+		spawn(20)
+			// This delay is necessary because /obj/structure/blob/core/proc/create_overmind has a spawn() in it which causes core.overmind not to be defined immediately.
+			if(core.overmind)
+				core.overmind.add_points(60)

```

Comment:
This gives lateround (and only lateround) blobs 60 points to start with. This replaces the fact that the old blob event tick() called process() every 3, which with 120 cycles of the event to run, would process() ~40 times. Each process() seems to add more than one resource... so blobs might get 80 or more resources from the old way this was done. Still, I'd prefer to err on the side of caution here, so I only give them a flat 60 resources. I also consider this better as it prevents blobs who spawn in a bad position from being instakilled with no hope of even defending themselves due to starting with no resources.